py/client-ticking/setup.py

```diff
@@ -29,6 +30,27 @@ def _compute_version():
     return _normalize_version(os.environ['DEEPHAVEN_VERSION'])
 
 _version = _compute_version()
+_system = platform.system()
+
+if _system == 'Windows':
+    dhinstall=os.environ.get('DHINSTALL')
```

Comment:
Should we warn if `dhinstall` is None or empty?

Reply:
Good catch.  I think just failing (by moving from `.get` to `[]` to access the environ dict) is better, also more consistent with what the script was already doing for other environment variables.